### PR TITLE
(BKR-562) removed leftover spec lines

### PR DESCRIPTION
**Pull request by kevpl**

### CODE REVIEW

File: spec/beaker-hiera/helpers_spec.rb [hunk 1]
```diff
@@ -11,18 +11,8 @@ def logger
...
-
-  let( :master ) { make_host( 'master',   :roles => %w( master agent default)    ) }
-  let( :agent )  { make_host( 'agent',    :roles => %w( agent )           ) }
-  let( :custom ) { make_host( 'custom',   :roles => %w( custom agent )    ) }
-  let( :dash )   { make_host( 'console',  :roles => %w( dashboard agent ) ) }
-  let( :db )     { make_host( 'db',       :roles => %w( database agent )  ) }
-  let( :hosts )  { [ master, agent, dash, db, custom ] }
-
+  let( :host  ) { make_host( 'master', :roles => %w( master agent default) ) }
+  let( :hosts ) { [ host ] }
```
Comment (colinPL) [inline spec/beaker-hiera/helpers_spec.rb]: Any concerns about moving from a multi-element array to just one?

Comment (kevpl) [inline spec/beaker-hiera/helpers_spec.rb]: Not as far as I can see.  None of the tests are there to ensure that it either takes or returns an array of hosts, they just want to be sure to act on the default host.

I suspect that the array was just a holdover from copy-pasting from a more complex testing situation to get this one going.

**Comment by sschneid** [comment]
:+1:

**Comment by waynr** [comment]
:+1:

**Pull request merged**